terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/webdrivers/FirefoxDriverFactoryBean.java

```diff
@@ -13,7 +13,7 @@
  * either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-package org.terasoluna.gfw.functionaltest.app;
+package org.terasoluna.gfw.functionaltest.app.webdrivers;
```

Comment:
- [ ] Unnecessary changes. (5.4.x needs only firefox factory bean.)

Reply:
fixed by the next commit.
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/806/commits/af8a63b38e21e4f50b51fe8110901876c485f5b4